src/hooks/useFocusRef.tsx

```diff
@@ -0,0 +1,11 @@
+import { useRef, useLayoutEffect } from 'react';
+
+export function useFocusRef<T extends HTMLOrSVGElement>(isCellSelected: boolean | undefined) {
```

Comment:
If we ever decide to export it out of rdg, we might want to consider a less generic name.